Mathlib/Tactic/Linter/TextBased.lean

```diff
@@ -349,12 +427,290 @@ def isImportsOnlyFile (lines : Array String) : Bool :=
 
 end
 
+
+namespace unicodeLinter
```

Comment:
This is really quite long; can it go in its own file?

Reply:
absolutely! I'll move it to `Mathlib/Tactic/Linter/Unicode.lean`